src/utils/index.ts

```diff
@@ -129,11 +125,7 @@ export function formatData(data: Data, args: Inputs): Output {
   // Lastly, all remaining arguments are forwarded as separate HTML attributes
   const htmlAttrInputs = filterArgs(
     args,
-    [
-      ...Object.keys(scriptUrlParamInputs),
-      ...Object.keys(htmlUrlParamInputs),
-      ...Object.keys(htmlSlugParamInput),
-    ],
+    [...Object.keys(htmlUrlParamInputs), ...Object.keys(htmlSlugParamInput)],
```

Comment:
removed the script inputs, don't think we need those, since those "should" be separate from html inputs